### ISSUE TITLE
Ctrl+Shift+V paste broken in 0.7

### ISSUE THREAD

**Issue by alexwhitman** [Guake/guake#534]
Pasting with Ctrl+Shift+V no longer pastes but instead just prints `^V` to the terminal.  Shift+Insert still works but as there are two clipboards it's not ideal.  Working around with the mouse/menu at the moment.

**Comment by Stibbons** [comment]
that's weird I use it pretty often

**Comment by alexwhitman** [comment]
Rolled back to 0.5 and then reinstalled 0.7 and it's working now.  Not sure what happened with it.

**Comment by lifeofguenter** [comment]
having the same problem - ctrl + shift + c/v do nothing.

**Comment by Stibbons** [comment]
It should be a deployment or installation issue. Can you uninstall + reinstall and relog to your account, to be sure the gconf shema has been properly installed?

**Comment by gcushen** [comment]
The guake currently available in Ubuntu Software Centre for Vivid is *0.5.0-2*. After installing, I discovered this is a very old version, so added the WebUpd8 PPA to get the latest. Then I realized that ctrl + shift + c/v do nothing as others have stated here. This is probably a very common install/upgrade use case. I can confirm the solution is, as per @Stibbons advice:
    
    sudo apt-get purge guake
    sudo apt-get install guake
    gnome-session-quit
